### PR TITLE
Release notes for Docker Desktop 4.22.1

### PR DESCRIPTION
**Pull request by chaizeg**
### Proposed changes

Add release notes for Docker Desktop 4.22.1

### CODE REVIEW

File: content/desktop/release-notes.md [hunk 1]
```diff
@@ -24,11 +24,27 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+
+### Bug fixes and enhancements
+
+#### For all platforms
+
+- Mitigated Docker Desktop startup issues, both during initial start as well as on exit from resource saver mode. [docker/for-mac#6933](https://github.com/docker/for-mac/issues/6933)
+
+#### For Windows
+
+- Fixed "Clean / Purge data" on Windows. [docker/for-win#13630](https://github.com/docker/for-win/issues/13630)
```
Comment (aevesdocker) [inline content/desktop/release-notes.md:41]:
```suggestion
- Fixed `Clean / Purge data` troubleshoot option on Windows. [docker/for-win#13630](https://github.com/docker/for-win/issues/13630)
```

File: content/desktop/release-notes.md [hunk 2]
```diff
@@ -24,11 +24,27 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+
+{{< release-date date="2023-08-24" >}}
+
+{{< desktop-install all=true version="4.22.1" build_path="/" >}}
+
+### Bug fixes and enhancements
+
+#### For all platforms
+
+- Mitigated Docker Desktop startup issues, both during initial start as well as on exit from resource saver mode. [docker/for-mac#6933](https://github.com/docker/for-mac/issues/6933)
```
Comment (aevesdocker) [inline content/desktop/release-notes.md:37]:
```suggestion
- Mitigated Docker Desktop startup issues, both during initial start as well as on exit from Resource Saver mode. [docker/for-mac#6933](https://github.com/docker/for-mac/issues/6933)
```

**Review by aevesdocker — APPROVED** [review]
⭐

**Review by aevesdocker — APPROVED** [review]

**Review by bsousaa — APPROVED** [review]
LGTM

**Pull request merged**